test/qunit/model/FHIRModel.unit.js

```diff
@@ -1161,4 +1161,15 @@ sap.ui.define([
 		assert.strictEqual(oResource.id, "254", "Response with location without having / at the beginning gives proper resource object and doesnot throw error");
 	});
 
+	QUnit.test("Search Bundle Respone Shouldnot Throw Error If Resource Id is not present ", function(assert) {
```

Comment:
[style] "Search Bundle Response should not throw an error if ResourceId is not present"

Reply:
done